### ISSUE TITLE
Scraping error: login.app-setup.token

### ISSUE THREAD

**Issue by chilek** [jwilk/mbank-cli#76]
```
Scraping error: login.app-setup.token: r02d8QOavePCA05I1wimB7PPQiDZvsgCz+5bzZcp5bIBy1PaMLdyCZWEprPxaKLKI13NmyZO404Z8172lPDAduRnkYn4o5lsIjniQzpK3yB6HGSOOYuwj95PGO3awEtloKQhgEmRaMJnTsKEu4MvnrzeGW5mZhFK3etLJyDGiI2nl1Nq6yby67f+MWdM+4znl4mXV/NDscGYsUOpnQSd3/C4G0IGbCcxluujCV6kMHl6bkrHAZGAGzPxdDk= at /opt/chilan/mbank-cli line 1175.
	main::do_login() called at /opt/chilan/mbank-cli line 2796
```

**Comment by jwilk** [comment]
Does 201a70edbf540ef9b92720f6bcc51b3b9bdbd357 help?

**Comment by chilek** [comment]
```perl
#!/usr/bin/perl

$token = "7rKCXqf38xGRhus1DTOsUbd+/iErR8ZgtASmIxBr4K8P0Jl16v/mOTtXYQZML1KJlMAZNVX0ZJNvjI4+ELQx9jDYamYc4znVmZfny7aA1K28biposvFMkacAFm8TUBJZZNIxI2905HJQz/hskaqyoDJIMCZpJnIq8baXg2YDM8dEUpOr2Xaz9535YKFzA59yvXJ/Os5hqQ2n3C2Krw9zwFWt83xh6BYsnHcpPCvvAZq45ryJMtHwbSVbJVs=";
$token =~ m{\A[\w+/]+\z}
    or print STDERR "old regexp: error\n";
$token =~ m{\A[\w+/=]+\z}
    or print STDERR "new regexp: error\n";
```

Result:
```
old regexp: error
```

Thanks a lot!

**Comment by jwilk** [comment]
Let's keep the issue open until it's fixed in a release. ☺

**Comment by chilek** [comment]
Ok, sorry.

**Comment by jwilk** [comment]
Fixed in [mbank-cli 2.1](https://github.com/jwilk/mbank-cli/releases/tag/2.1).
